lib/node_modules/@stdlib/stats/incr/prod/lib/main.js

```diff
@@ -86,7 +86,7 @@ function incrprod() {
 			return ( prod === void 0 ) ? null : prod;
 		}
 		// Split the incoming value into a normalized fraction and exponent:
-		frexp( PARTS, x );
+		frexp.assign( x, PARTS );
```

Comment:
```suggestion
		frexp.assign( x, PARTS, 1, 0 );
```

Missing stride and offset.